plugins/generic/externalFeed/templates/editExternalFeedForm.tpl

```diff
@@ -0,0 +1,88 @@
+{**
+ * plugins/generic/externalFeed/templates/editExternalFeedForm.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Form for editing a external feed
+ *}
+ 
+<script type="text/javascript">
+	$(function() {ldelim}
+		// Attach the form handler.
+		$('#externalFeedForm').pkpHandler(
+			'$.pkp.controllers.form.AjaxFormHandler'
+		);
+	{rdelim});
+</script>
+
+{url|assign:actionUrl router=$smarty.const.ROUTE_COMPONENT component="plugins.generic.externalFeed.controllers.grid.ExternalFeedGridHandler" op="updateExternalFeed" existingPageName=$blockName escape=false}
+<form id="externalFeedForm" class="pkp_form" method="post" action="{$actionUrl}">
+	{csrf}
+	
+	{if $feedId}
+		<input type="hidden" name="feedId" value="{$feedId|escape}" />
+	{/if}
+	
+	{include file="controllers/notification/inPlaceNotification.tpl" notificationId="ExternalFeedFormNotification"}
+	
+	{fbvFormSection}
+		{fbvFormSection for="title " title="plugins.generic.externalFeed.form.title" description="plugins.generic.externalFeed.form.titleDescription"}
+			{assign var="formLocale" value=$formLocale|escape}
+			{fbvElement type="text" id="title" name=title[$formLocale] value=$title|escape}

```

Comment:
No need to `escape` the value when using `fbvElement` -- this could lead to double-escaping when special characters are included in the form element value.